src/components/Task/Drawer/Form/Form.stories.tsx

```diff
@@ -19,6 +19,17 @@ export default {
   title: 'Task/Drawer/Form',
 };
 
+const mockFilter = {
```

Comment:
Also not sure what's the best way to approach where this mockFIlter lives. Should I just put it in every file that needs it, or should I create a separate file for it and just import that into a test or storybook if it needs it? I will also have different variations of the filter(ex. ```emptyMockFilter```, ```mockFilter```, etc).

Reply:
🤷‍♂️ You could export it.. Idk where would be best. Maybe the test?